### ISSUE TITLE
Metrics definition

### ISSUE THREAD

**Issue by IssaIssa1** [google-research/electra#81]
Hello, may someone please explain what does each evaluation metric indicates.

```
  disc_accuracy = 0.86676794
  disc_auc = 0.6815034
  disc_loss = 0.35936752
  disc_precision = 0.7586109
  disc_recall = 0.04827826
  global_step = 5000
  loss = 25.108772
  masked_lm_accuracy = 0.032289855
  masked_lm_loss = 7.1341
  sampled_masked_lm_accuracy = 0.0033757575
```
